src/main/java/uk/gov/companieshouse/api/accounts/controller/LoansToDirectorsAdditionalInformationController.java

```diff
@@ -69,6 +83,11 @@ public ResponseEntity update(@RequestBody AdditionalInformation additionalInform
             return new ResponseEntity<>(HttpStatus.NOT_FOUND);
         }
 
+        if (bindingResult.hasErrors()) {
```

Comment:
Would you do this before looking up the LoansToDirectors links?

Reply:
I considered this, but I think regardless of whether data submitted is valid or not we ought to check a resource exists first. Imo, if a user were to PUT bad data against a non-existent resource, a `404` would trump a `400` but I'm happy to defer to the team if they have stronger feelings either way